### PR TITLE
Add resort page and React Ruter

### PR DESCRIPTION
**Pull request by dorkka**
Add hardcoded weather info for resorts to db.json, add SpecificResortPage with details and weather

https://trello.com/c/R0ZgsdcQ/17-use-react-router-for-routing-instead-of-home-made-routing-solution

### CODE REVIEW

File: src/components/resortPage/SpecificResortPage.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    };
+  }
+  
+  componentDidMount(){
+    this.fetchResort();
+  }
+
+  fetchResort(){
+    this.setState({isLoading: true})
+    fetch(`http://localhost:3001/resorts/${this.state.resortId}`)
```
Comment (ar-mac) [inline src/components/resortPage/SpecificResortPage.js]: note: refactor this to HOC

File: src/components/resortPage/ResortWeatherInfo.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+class ResortWeatherInfo extends Component {
+  render() {
+    return (
+      <div className="col-md-4">
+        <h4>Pogoda</h4>
+        <p>Temperatura: {this.props.temperature}</p>
```
Comment (ar-mac) [inline src/components/resortPage/ResortWeatherInfo.js]: note: use destructuring and functional components?

**Comment by pokiujf** [comment]
Check code auto styling in VSCode

**Comment by pokiujf** [comment]
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import

File: src/components/ResortsData.js [hunk 3]
```diff
@@ -2,21 +2,22 @@ import React, { Component } from 'react';
 import ResortBasicInfo from './ResortBasicInfo';
 import ResortsListHead from './ResortsListHead';
 import Pagination from './Pagination';
+import resourceFetcher from '../services/resourceFetcher';
 
-class ResortsData extends Component{
-  constructor(){
+class ResortsData extends Component {
+  constructor() {
```
Comment (ar-mac) [inline src/components/ResortsData.js]: No need for constructor. Define `state` as class property

File: src/services/resourceFetcher.js [hunk 4]
```diff
@@ -0,0 +1,12 @@
+import qs from 'qs';
+
+export default url => (params) => {
```
Comment (ar-mac) [inline src/services/resourceFetcher.js]: (minor) 
`export default resourceFetcher`

File: src/components/resortPage/SpecificResortPage.js [hunk 5]
```diff
@@ -0,0 +1,73 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import ResortWeatherInfo from './ResortWeatherInfo';
+import ResortDetails from './ResortDetails';
+
+
```
Comment (ar-mac) [inline src/components/resortPage/SpecificResortPage.js]: ✂️

**Review by ar-mac — CHANGES_REQUESTED** [review]

**Comment by ar-mac** [comment]
[Use React-Router for routing instead of home made routing solution](https://trello.com/c/R0ZgsdcQ/17-use-react-router-for-routing-instead-of-home-made-routing-solution)

File: src/components/resortPage/SpecificResortPage.js [hunk 6]
```diff
@@ -23,7 +23,11 @@ class SpecificResortPage extends Component {
         if (response.ok) {
           return response.json();
         }
-        throw new Error('Something went wrong ...');
+        let message = 'Something went wrong ...';
```
Comment (ar-mac) [inline src/components/resortPage/SpecificResortPage.js]: - This should be done only in resourceFetcher
- Use Resource fetcher here
- In state set only resort and isLoading

**Review by ar-mac — APPROVED** [review]

**Pull request merged**